setup.cfg

```diff
@@ -36,7 +36,24 @@ install_requires =
 setup_requires =
     setuptools >= 41.2
     setuptools_scm
-    
+
+[options.extras_require]
+# For plink we need dask[dataframe], we already have
+# dask[array] in install_requires, and since
+# https://github.com/pypa/pip/issues/4957, pip
+# will essentially ignore dask[dataframe] in the extras.
+# We can workaround this by either adding pip flag
+# --use-feature 2020-resolver, or installing
+# dask[dataframe] in the install_requires, or just listing
+# the 2 missing dependencies from dataframe, the way we do
+# here, when pip finally gets a resolver, this won't be
+# a problem. Here we opt for listing the 2 dependencies
+# since this is the least user invasive solution.
+plink =
+    partd
+    fsspec
+    bed-reader
```

Comment:
I can't see where dask[dataframe] actually gets installed - it looks like only its dependencies get installed here?

Reply:
@tomwhite see the comment above (https://github.com/pystatgen/sgkit/pull/277/files#diff-380c6a8ebbbce17d55d50ef17d3cf906R41-R51) for the context. Does that make sense?

Reply:
It makes sense, but I'm still missing something: where is the `dask[dataframe]` dependency declared? It's used by `pysnptools.py`, but how does it get installed?

Reply:
`dask[dataframe]` is not an "actual" dependency, it's an extra in `dask`, and since we already install `dask` via `dask[array]` in `install_requires` and since it's an **extra in our extras** (which [this bug](https://github.com/pypa/pip/issues/4957) in pip affects) we can't really list `dask[dataframe]` as a dependency (unless we use the 2020 pip resolver), so we have the options outlined in the comment above, so instead of forcing the extra flag on users, I opts for listing the missing dependencies that come from `dask[dataframe]`. Initial version of this PR actually used the 2020 resolver, but I reverted to listing `dask[dataframe]` deps directly so that we can use plain `pip`.

Reply:
Ah, got it. I was missing the bit about `dask[dataframe]` not actually providing any Dask code.

When the pip bug is fixed do you think we should switch to just add the `dask[dataframe]` dependency here?

Reply:
@tomwhite quick answer - sure. But that bug will be fixed by the 2020 resolver, right now 2020 resolver is available via feature flag, it is scheduled to be default in October this year, so we can wait until then + a couple of months and we should be good to switch over.